EventFilter/RPCRawToDigi/interface/RPCLBRecord.h

```diff
@@ -0,0 +1,71 @@
+#ifndef EventFilter_RPCRawToDigi_RPCLBRecord_h
+#define EventFilter_RPCRawToDigi_RPCLBRecord_h
+
+#include <stdint.h>
```

Comment:
As far as I know, `<cstdint>` is not required to declare these outside the `std` namespace though.